### DISCUSSION TITLE
Principal Stress

### DISCUSSION THREAD

**Discussion by ncarmona8** [KratosMultiphysics/Kratos#9834] [Q&A]
Hi... I am creating a model for an undergrpund galery to study stress conditios... In my post process i just can see von misses stress, but, i need to get the valuoes for the principal stress conditions and cauchy tensor....
I hope you can help me

**Comment by RiccardoRossi** [comment]
you should be able to plot the CAUCHY_STRESS (well, need to check the exact
name of the variable) on the gauss points and then visualize it in gid

**Reply by ncarmona8** [reply]
Disculpa esto puede configurarse solo trabajando desde la interfaz de GiD?

**Reply by ncarmona8** [reply]
What you refeer on the gauss points?? i only have worked my model in GiD Interface

**Reply by rubenzorrilla** [reply]
What @RiccardoRossi meant is to activate the Gauss point magnitudes output in the GiD output process. This can be done either in the interface (Results -> On element --> Cauchy stress tensor) or by directly setting the `CAUCHY_STRESS_TENSOR` variable in the ProjectParameters.json output settings (e.g. `"gauss_point_results" : ["VON_MISES_STRESS","CAUCHY_STRESS_TENSOR"]`).

**Reply by ncarmona8** [reply]
Thank you very much
